www/templates/default/html/manager/AddDatetime.tpl.php

```diff
@@ -130,74 +135,92 @@
         <label for="room">Room</label>
         <input type="text" id="room" name="room" value="<?php echo $datetime->room; ?>" />
 
+        <?php #Don't show the box when the current eventdatetime is a recurring instance or current evendatetime is a recurring and multi-day type.
+            if ($context->recurrence_id == NULL && !($datetime->recurringtype != 'none' && $event_duration_type == 'multi_day')) : ?>
+            <label for="single-day-event"><span class="required">*</span> Event duration type</label><br>
+            <input <?php if ($event_duration_type =='single_day') echo 'checked=checked' ?> id="single-day-event" type="radio" value="single_day" name="event_duration_type">
+            <label>Single Day</label>&nbsp;
```

Comment:
Labels must be programmatically associated with controls for accessibility. This is the only way the label will be announced by assistive technology such as screen readers.

Options include:
* reference the control via the `id` attribute (explicit association)
* wrap the control in the <label> (implicit association)